Formula/inko.rb

```diff
@@ -13,20 +13,18 @@ class Inko < Formula
     sha256 "49de93ab54879a2a48bc4e3dce4f2bc52a5912085583de9290895eadae4b119e" => :high_sierra
   end
 
-  depends_on "autoconf" => :build
-  depends_on "automake" => :build
   depends_on "coreutils" => :build
-  depends_on "libtool" => :build
   depends_on "make" => :build
   depends_on "rust" => :build
+  depends_on "libffi"
```

Comment:
I think macOS ships this right? Because that means it can be `uses_from_macos`

Reply:
@SMillerDev I don't think libffi is shipped with macOS, unless that's a recent addition.